resources/fetchers/network_fetcher.go

```diff
@@ -39,24 +39,28 @@ type ACLFetcherConfig struct {
 	fetching.AwsBaseFetcherConfig `config:",inline"`
 }
 
-type ACLResource struct {
+type NetworkResource struct {
 	awslib.AwsResource
 	identity *awslib.Identity
 }
 
-// Fetch collects ACL resources
-// The resources are enriched by the provider and being send to evaluation.
-func (f ACLFetcher) Fetch(ctx context.Context, cMetadata fetching.CycleMetadata) error {
-	f.log.Debug("Starting ACLFetcher.Fetch")
+// Fetch collects network resource such as network acl and security groups
+func (f NetworkFetcher) Fetch(ctx context.Context, cMetadata fetching.CycleMetadata) error {
+	f.log.Debug("Starting NetworkFetcher.Fetch")
 
-	resources, err := f.aclProvider.DescribeNeworkAcl(ctx)
+	nacl, err := f.aclProvider.DescribeNeworkAcl(ctx)
 	if err != nil {
 		return err
```

Comment:
I'm not sure about this behavior, in case we fail to retrieve a certain resource do we want to return an error and stop fetching the rest of the network resources? 

In the `IAM fetcher`,  I continue with the fetching cycle even if some error occurred, anyway whatever we decide -the logic should be the same.

Reply:
Sounds right, will change that